airflow/providers/apache/hive/hooks/hive.py

```diff
@@ -78,9 +78,13 @@ class HiveCliHook(BaseHook):
     :type  mapred_job_name: str
     """
 
+    conn_name_attr = 'hive_cli_conn_id'
```

Comment:
What was your thinking for `conn_name_attr` instead of `CONN_NAME_ATTR`?

(My default would be to use the latter, as it's a common signifier that it's a constant.)

Reply:
Just following "do the same as others"  simply we already had other connection class attributes lowercase (convention used in the SQL Hooks). So I just followed it.

Reply:
I'm not sure the convention from SQL hooks is the best one -- those are some of the oldest parts of Airflow and not very "Good Practice".

I'm okay with it if you think this way is better, but may other places in Airflow we have UPPERCASE style.

Reply:
I fully agree. But I did not want to break the compatibility (potentially someone could have used those values - especially the defaults. And my goal was just to make it works for providers, not having to deal with back-compatibility :). 

There is a polish sentence I tend to use in such cases which I learned from my friend (but it has some curse-words and rhymes, so I won't translate it here verbatim, but it roughly translates to "consistent is better than perfect".

I fully agree constants *should* be UPPERCASE of course.

Reply:
BTW. Happy to make a change later (maybe even we manage in 2.0 if you think it's ok to break compatibility with the names. Once it is consistent, we can do global rename.